src/linter/root.go

```diff
@@ -717,6 +726,17 @@ func (d *RootWalker) enterPropertyList(pl *stmt.PropertyList) bool {
 	for _, pNode := range pl.Properties {
 		p := pNode.(*stmt.Property)
 
+		switch exprProperty := p.Expr.(type) {
```

Comment:
Why not just do node.Walk() here too, as you did with the constant list? I can't say I like the code duplication here...